src/Presenter/PsBillingPresenter.php

```diff
@@ -84,32 +74,36 @@ public function __construct(
      */
     public function present(array $params)
     {
-        $getEnv = !empty($params['billingEnv']) ? $params['billingEnv'] : '';
-        $billingEnv = $this->envBuilder->buildBillingEnv($getEnv);
+        $getEnv = $this->getBillingAccountsWrapper()->getBillingEnv()
```

Comment:
PHP >= 5.3

`$getEnv = $this->getBillingAccountsWrapper()->getBillingEnv() ?: '';`

Reply:
ah je connaissais pas cette notation

Reply:
C'est l'équivalant de ?? en JS ;)